### PR TITLE
feat(page-cluster): init page

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-500)

- Added Cluster detail page with Settings navigation (test [here](http://localhost:4200/organization/141c07c8-0dd9-4623-983b-3fdd61867255/cluster/a8ad0659-b8f2-4c83-ad77-092e97bb2cae/settings/general))
- Missing Breadcrumb improvement with Clusters
- And, not connected with the Clusters page because we don't have settings for now

Capture d’écran 2023-01-17 à 11 33 02

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(page-cluster): init page - 2023-01-16T17:27:29Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/635358ee-3e36-42cc-a18d-4b590eaa9590/applications)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment failed to be deployed !
Click on the link below to check the deployment logs.
:point_right: [[PR] staging - feat(page-cluster): init page - 2023-01-16T17:27:29Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/635358ee-3e36-42cc-a18d-4b590eaa9590/applications?fullscreenLogs=true)

File: libs/pages/cluster/src/lib/page-cluster.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// import { render } from '__tests__/utils/setup-jest'
+// import PagesCluster from './page-cluster'
+
+describe('PagesCluster', () => {
+  it('should render successfully', () => {
+    // const { baseElement } = render(<PagesCluster />)
```
Comment (bdebon) [inline libs/pages/cluster/src/lib/page-cluster.spec.tsx:6]: Why is this commented?

**Review by bdebon — APPROVED** [review]
lgtm except for some comments in a test file

**Pull request merged**